subgraphs/aave-v2/src/modules/Liquidate.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  getProtocolIdFromCtx,
+  getOrCreateLendingProtocol,
+  getOrCreateMarketDailySnapshot,
+  getOrCreateMarketHourlySnapshot,
+  getOrCreateFinancialsDailySnapshot,
+  getOrCreateUsageMetricsDailySnapshot,
+  getOrCreateUsageMetricsHourlySnapshot,
+} from "../common/initializers";
+import * as constants from "../common/constants";
+import { getOrCreateToken } from "../common/initializers";
+import { Liquidate as LiquidateEntity, Market } from "../../generated/schema";
+import { Address, BigDecimal, BigInt, ethereum } from "@graphprotocol/graph-ts";
+
+export function createLiquidateEntity(
+  event: ethereum.Event,
+  market: Market,
+  debtAsset: string,
+  collateralAsset: string,
+  liquidator: string,
+  amount: BigInt
+): void {
+  const protocolId = getProtocolIdFromCtx();
+  const protocol = getOrCreateLendingProtocol(protocolId);
+  const inputToken = getOrCreateToken(Address.fromString(collateralAsset));
+
+  const id: string = `${event.transaction.hash.toHexString()}-${event.logIndex.toString()}`;
+  const liquidate = new LiquidateEntity(id);
+
+  liquidate.to = debtAsset;
+  liquidate.market = market.id;
+  liquidate.from = liquidator;
+  liquidate.hash = event.transaction.hash.toHexString();
+  liquidate.logIndex = event.logIndex.toI32();
+  liquidate.protocol = protocol.id;
+  liquidate.asset = collateralAsset;
+  liquidate.amount = amount;
+  liquidate.liquidatee = event.transaction.from.toHexString();

```

Comment:
from my understanding the liquidatee is this address: 
![image](https://user-images.githubusercontent.com/60583747/175834532-0a00ac67-cb15-42fd-b4c7-0348bee60216.png)
https://etherscan.io/tx/0xf712b360811a8e66c78437417f514643c0123e58db6d310d2a16000ccf057306#eventlog